lib/cli.js

```diff
@@ -82,7 +83,7 @@ function printResults(engine, results, format, outputFile) {
         if (outputFile) {
             const filePath = path.resolve(process.cwd(), outputFile);
 
-            if (fs.existsSync(filePath) && fs.statSync(filePath).isDirectory()) {
+            if (pathUtil.isDirectory(filePath)) {
```

Comment:
Good catch.
If I replace pathUtil by `shell.test("-d"` and `shell.test("-f"`, I don't need to add new functions.